engine/orchestration/src/com/cloud/vm/VirtualMachinePowerStateSyncImpl.java

```diff
@@ -107,7 +104,7 @@ private void processReport(long hostId, Map<Long, VirtualMachine.PowerState> tra
                 s_logger.debug("Run missing VM report. current time: " + currentTime.getTime());
 
             // 2 times of sync-update interval for graceful period
-            long milliSecondsGracefullPeriod = PingInterval.value() * 2000L;
+            long milliSecondsGracefullPeriod = ManagementServiceConfiguration.PingInterval.value() * 2000L;
```

Comment:
Since you've introduced a new `ManagementServiceConfiguration`, why not consume the value using a method?